airflow/jobs.py

```diff
@@ -1790,6 +1790,56 @@ class BackfillJob(BaseJob):
         'polymorphic_identity': 'BackfillJob'
     }
 
+    class _DagRunTaskStatus(object):
+        """
+        Internal status of the backfill job. This class is intended to be instantiated
+        only within a BackfillJob instance and will track the execution of tasks,
+        e.g. started, skipped, succeeded, failed, etc. Information about the dag runs
+        related to the backfill job are also being tracked in this structure,
+        .e.g finished runs, etc. Any other status related information related to the
+        execution of dag runs / tasks can be included in this structure since it makes
+        it easier to pass it around.
+        """
+
+        # TODO(edgarRd): https://issues.apache.org/jira/browse/AIRFLOW-1444
```

Comment:
Nit: let's kill the whitespace/newlines, make it more descriptive (e.g. summary of the JIRA), and no need to include the full JIRA URL, just AIRFLOW-144 is OK (consistent with the other JIRA TODOs in the project)